src/subscription_manager/repolib.py

```diff
@@ -167,6 +174,11 @@ def enable_pkg_plugins(cls):
         :return: It returns list of enabled plugins
         """
 
+        d_name, d_version, d_id, vers_mod, id, id_like = HardwareCollector().get_distribution()
```

Comment:
Since you need to unpack only two values of the tuple returned by `get_distributor()`, better assign it to a single variable; see what `HardwareCollector.get_release_info()` does.